jonprl-mode.el

```diff
@@ -85,11 +85,11 @@ manually."
   :group 'jonprl
   :options '(jonprl-update-operators))
 
-(defcustom jonprl-pre-check-buffer-hook '(save-buffer)
+(defcustom jonprl-pre-check-buffer-hook '(save-buffer jonprl-initialize-configuration)
   "A hook to run prior to checking the buffer."
   :type 'hook
   :group 'jonprl
-  :options '(save-buffer))
+  :options '(save-buffer jonprl-initialize-configuration))
```

Comment:
Well, a user may wish to disable that feature if it irritated them... But I think that's not necessarily a use-case I feel the need to support.

I'll move it as you suggest.